src/ucp/core/ucp_mm.c

```diff
@@ -150,9 +151,8 @@ static ucs_status_t ucp_mem_alloc(ucp_context_h context, size_t length,
             }
         }
 
-        /* TODO: pass addr */
-        status = uct_mem_alloc(NULL, length, uct_flags, &method, 1, mds, num_mds,
-                               name, &mem);
+        status = uct_mem_alloc(memh->address, length, uct_flags, &method, 1, mds,
```

Comment:
I agree with the idea of following mmap semantics. If UCP_MEM_MAP_FIXED was specified UCP should fail and return an error.